### DISCUSSION TITLE
Inventory: Finished Product/Components

### DISCUSSION THREAD

**Discussion by WebSpectrum** [timschofield/webERP#319] [Feature Requests]
Does inventory allow for costing and tracking of finished products & sub products i.e. components?

**Comment by dalers** [comment]
Hi @WebSpectrum. For purchased items, webERP supports Standard Costing and Weighted Average Costing.

Manufactured Items and Assembly Items (aka a "kit") have a BIll of Materials (BOM). The cost of a manufactured item is the sum of the items on the BOM (and optionally a labor cost and an overhead cost) and is updated automatically when a work order is created to manufacture the item. The cost of an assembly item is updated when a sale is invoiced.

Does that answer your question? If not, could you please describe your situation and what behavior you are expecting.

Cheers,
Dale
